source/validate_id.cpp

```diff
@@ -740,7 +740,7 @@ bool idUsage::isValid<SpvOpLoad>(const spv_instruction_t* inst,
            return false);
   auto pointerIndex = 3;
   auto pointer = usedefs_.FindDef(inst->words[pointerIndex]);
-  if (!pointer.first || !spvOpcodeIsPointer(pointer.second.opcode)) {
+  if (!pointer.first || !spvOpcodeIsPotentialPointer(pointer.second.opcode)) {
```

Comment:
Actually, the Vulkan spec [does say](https://www.khronos.org/registry/vulkan/specs/1.0/xhtml/vkspec.html#spirvenv-module-validation) that Logical must be selected for graphics modules.